### DISCUSSION TITLE
Not running under Windows 10 pro

### DISCUSSION THREAD

**Discussion by Brayo254** [pawelsalawa/sqlitestudio#3436] [Q&A]
### Details
Hello, I am using windows 10 pro. I recently downloaded sqlite studio from https://sqlitestudio.pl/index.rvt?act=download but it doesnt load. The application is installed but when launching nothing is happening. No error just nothing. It is worth noting i have MSQL and XAMPP installed on my machine. Although they work simultaneously because i configured them to do so. So why is sqlite not launching at all after installation of the .exe file?

**Comment by pawelsalawa** [comment]
Hit Left_Ctrl + Left_Shift + Esc. Task manager will open. Switch to details tab, sort the list alphabetically and then look for SQLiteStudio.exe process (after you run SQLiteStudio but it did not start). Is it there or not?

**Comment by Brayo254** [comment]
Yes I saw it when I immediately running sqlite studio but it vanished. It
was under "details"

**Comment by pawelsalawa** [comment]
Try this:

- Create directory C:\log
- Hit Win+r, type "cmd" and enter. Command line will open.
- There use "cd" command to enter the directory where SQLiteStudio is installed (most likely C:\Program Files\SQLiteStudio).
- type: sqlitestudio.exe -d --debug-file C:\log\log.txt

After a moment please look into the log file. Maybe you will find solution from logs by yourself. If not, please paste the log here and I will look into it.

**Comment by Brayo254** [comment]
Ok. But before I do that I should let u know that I booted in safe mode and
it works perfectly. But I need it to run like any other application. I am
surprised.

**Comment by Brayo254** [comment]
![screenshot_20181017-184258](https://user-images.githubusercontent.com/24669288/47099363-cd0fc180-d23d-11e8-96d5-5d519ded7044.png)

I did as you asked but still nothing in the "Log" folder. See attached pic.

**Comment by pawelsalawa** [comment]
Okay, let's try one more thing. Try downloading portable distribution (not the exe, but zip file), unzip it and try running from there.

**Comment by vfbear** [comment]
Hello, i met the same issue. If running SQLiteStudio.exe by click or cmd, it will appear nothing, and with exception code 0xc0000005 in windows app log. But by PowerShell cmd, it will be OK. Hope it's helpful.

**Comment by pawelsalawa** [comment]
Do you have any antivirus (other than built-in Windows Defender) installed? If so, try with it disabled for a moment.

**Comment by pawelsalawa** [comment]
You can also try downloading some older version of SQLiteStudio and check with that. For example https://sqlitestudio.pl/files/sqlitestudio3/complete/win32/sqlitestudio-3.1.1.zip or even https://sqlitestudio.pl/files/sqlitestudio3/complete/win32/sqlitestudio-3.0.7.zip

**Comment by Brayo254** [comment]
I wanna thank you all for those who took time to offer suggestions to my problem. Apparently running older version of SQLite(V3.0.7) 32bit helps. Thank you Paweł Salawa and every one else. Its now working.
Lets hope it behaves this time.

**Comment by vfbear** [comment]
V3.0.7 is normal, but V3.1.1 same as V3.2.1. I have only Windows Defender antivirus.

**Comment by Brayo254** [comment]
> V3.0.7 is normal, but V3.1.1 same as V3.2.1. I have only Windows Defender antivirus.

I use MalwareBytes. I had to temporarily stop it. Then ran the the application.

**Comment by pawelsalawa** [comment]
So with MalwareBytes disabled the 3.2.1 runs fine?

**Comment by Brayo254** [comment]
> So with MalwareBytes disabled the 3.2.1 runs fine?

No, version 3.0.7 runs fine. Although i will run with Malwarebytes on to see what happens.

**Comment by pawelsalawa** [comment]
Have you tried to run 3.2.1 with MalwerBytes turned off?

**Comment by Brayo254** [comment]
> Have you tried to run 3.2.1 with MalwerBytes turned off?

I will in a second.

**Comment by Brayo254** [comment]
![screenshot 54 _li](https://user-images.githubusercontent.com/24669288/47139987-cf1d6300-d2c5-11e8-917a-1c331b41dca8.jpg)

What the hell??????

**Comment by pawelsalawa** [comment]
Looks like a conflict of SQLiteStudio versions installed/running. Keep the one you installed in Program Files and remove others. Run the one from program files.

**Comment by Brayo254** [comment]
> Have you tried to run 3.2.1 with MalwerBytes turned off?

Its not running with malwarebytes off. BUT v 3.0.7 runs with malwarebytes on or off

**Comment by pawelsalawa** [comment]
Hm. Do you have Qt installed somewhere in your system?

**Comment by Brayo254** [comment]
> Hm. Do you have Qt installed somewhere in your system?

No I dont

**Comment by Brayo254** [comment]
> Looks like a conflict of SQLiteStudio versions installed/running. Keep the one you installed in Program Files and remove others. Run the one from program files.

OK let me try

**Comment by pawelsalawa** [comment]
I'm running out of ideas. This is really uncommon.

**Comment by Brayo254** [comment]
> I'm running out of ideas. This is really uncommon.

Me too. Because in program files there is still an error. The same i posted

**Comment by vfbear** [comment]
@Brayo254 Could you test V3.2.1 with PowerShell cmd? It's fine for me.

**Comment by Brayo254** [comment]
I finally solved it. Thanks to everyone.
